rules/dash-sai.mk

```diff
@@ -1,7 +1,7 @@
 # DASH SAI repo: https://github.com/sonic-net/DASH
 
 DASH_SAI_VERSION = 1.0.0
-DASH_SAI_COMMIT = cc281333e24ade258773ea843f669251a44d7c9a
+DASH_SAI_COMMIT = 88a4771
```

Comment:
do you mind to switch to a full hash?

Reply:
nothing particular, just more stable. : D

Reply:
fixed